source/Assets/Plugins/Didomi/Tests/GetUserStatusTestsSuite.cs

```diff
@@ -55,6 +55,8 @@ public IEnumerator TestUserStatusInfo()
         Assert.False(string.IsNullOrEmpty(userStatus.GetCreated()), "No created date");
         Assert.False(string.IsNullOrEmpty(userStatus.GetUpdated()), "No updated date");
         Assert.AreEqual("gdpr", userStatus.GetRegulation(), $"Incorrect regulation: {userStatus.GetRegulation()}");
+        // Assert.False(string.IsNullOrEmpty(userStatus.GetAdditionalConsent()), "No additional consent");
+        // Assert.False(string.IsNullOrEmpty(userStatus.GetDidomiDcs()), "No didomi DCS");
```

Comment:
If config is not specific to additional consent / DCS feature, I would expect the result to be True